catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -349,11 +249,21 @@ public CreateResponse create(CreateStorageRequest streamCreateRequest)
 
             createResponse = create(createRequest);
         } catch (Exception e) {
```

Comment:
I can specify the exceptions that are now thrown - `StorageException | IOException | RuntimeException` if that would be better.